lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -117,7 +137,7 @@ def endpoint
 
         def target
           default_target = "googleads.googleapis.com:443"
-          target = ENV.fetch('GOOGLEADS_SERVICE_PATH', default_target)
```

Comment:
You may want to keep the old one for a while and flag it as deprecated.

Reply:
Since this is an internal setting, external developers wouldn't use the old one any ways, I prefer replacing it rather than keeping it, unless there is strong reason not to do so. WDYT?

Reply:
SGTM.

Reply:
+1 on just changing it right away; Fei and I are the only users of this setting.